services/github/src/handlers.js

```diff
@@ -458,24 +458,24 @@ async function requestArtifact(artifactName, { taskId, runId, debug, instGithub,
     const limitedQueueClient = this.queueClient.use({
       authorizedScopes: scopes,
     });
-    const url = limitedQueueClient.buildUrl(limitedQueueClient.getArtifact, taskId, runId, artifactName);
+    const url = limitedQueueClient.buildSignedUrl(limitedQueueClient.getArtifact, taskId, runId, artifactName);
     const res = await utils.throttleRequest({ url, method: 'GET' });
 
     if (res.status >= 400 && res.status !== 404) {
       let errorMessage = "Failed to get your artifact.\n";
       switch (res.status) {
         case 403:
-          errorMessage.concat("Make sure your artifact is public. See the documentation on the artifact naming.");
+          errorMessage = errorMessage.concat("Make sure your artifact is public. See the documentation on the artifact naming.");
```

Comment:
Is this message still accurate?

Reply:
Ah, interesting point. I guess we could reword that to something like 'Make sure your artifact is accessible.'?

'Make sure you have assigned appropriate scopes to your task and repo roles.'?

I'm really not sold on either of those being amazingly easy to understand. :sweat_smile: 

Is it worth adding the artifact path to the first message? (That was something which wasn't obvious when I was trying to track the cause down here...)

Reply:
"Make sure your task has the scope `..`" might be the most explicit, and also would imply the artifact name.  I'll merge this fix, if you want to adjust the message in a followup?